### PR TITLE
CDAP-1961: Cube dataset APIs minor improvements

### PR DESCRIPTION
**Pull request by abaranau**
* CubeFact to contain multiple measurements
* CubeFact has nicer API (and closer to Table ops) to construct

### CODE REVIEW

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-DUT1456-2

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-DUT1456-3 GREEN

File: cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/cube/CubeFact.java [hunk 1]
```diff
@@ -29,38 +31,85 @@
...
-   * @param tagValues tag name, tag value pairs associated with the fact
-   * @param measureType measurement type, see {@link MeasureType} for available types
-   * @param measureName measurement name
-   * @param timeValue value of the measurement at specific time
+   * Creates an instance of {@link CubeFact} with no tags and no measurements.
+   * <p/>
+   * After creation, you can add tags e.g. via {@link #addTag(String, String)}
+   * and add measurements e.g. via {@link #addMeasurement(String, MeasureType, long)}.
+   *
+   * @param timestamp timestamp (epoch in sec) of the measurement
```
Comment (gsps1) [inline cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/cube/CubeFact.java]: measurements

**Comment by gsps1** [comment]
just one minor nit , LGTM :+1:

**Pull request merged**